style.css

```diff
@@ -198,3 +198,115 @@ section .circle2 {
 .portfolio .next .arrow-span {
   margin-left: 10px;
 }
+
+.recent-work {
+  padding: 30px;
+}

```

Comment:
- [ ] One of the requirements is to use CSS grid in your work section container, so please make sure to use `grid` to align your content inside.